straxen/plugins/peaklet_processing.py

```diff
@@ -499,6 +502,8 @@ def compute(self, peaklets_he):
 
 @export
 @strax.takes_config(
+    strax.Option('n_top_pmts', type=int,
+                 help='Number of PMTs in the top array'),
```

Comment:
Have you changed the option here by accident? At least the plugin itself stayed the same. Or I am blind, not unlikely on a Friday afternoon :D

Reply:
I think you're confusing n_TOP_pmts with n_TPC_pmts

Reply:
That was a mistake finding why merging crashed